src/app/navigation/navigation.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-navigation',
+  templateUrl: './navigation.component.html',
+  styleUrls: ['./navigation.component.scss']
```

Comment:
we shouldn't store empty files or files that do nothing. If you don't declare any styles at `navigation.component.scss` remove it